File: src/main/frontend/src/app/models/poDetails.ts
import { RmgTeam } from "./rmgTeam";
import { Team } from "./team";

export class PoDetails {

    id: any;
    poId: any;
    projectId: any;
    poNo: any;
    apmosysRM: any;
    clientRm: any;
    apmosysRmEmail: any;
    poStartDate: any;
    poEndDate: any;
    activeFlag: any;
    prevPO: any;
    nextPO: any;
    msg: any;
    clientLocationId: any;
    assigned: any;
    totalRequirements: any;
    assignedApproved: any;
    assignedPending: any;
    difference: any;
    newTeamObj: RmgTeam = new RmgTeam();
    teamList: RmgTeam[] = [];
    selectedTeamIds: any[] = [];
    isPoSelected: boolean = false;
    isAnyTeamSelected: boolean = false;
    isAnyNewTeamAdded: boolean = false;
    updatedBy: any;
    isHod: boolean = false;
    isupdate: boolean = true;
    projectType: any;

}

File: src/main/java/com/apmosys/employeeportal/model/PoDetails.java
package com.apmosys.employeeportal.model;

import java.time.LocalDateTime;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

import lombok.Data;

@Data
@Entity
public class PoDetails {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long poId;
    private Integer projectId;
    private String poNo;
    private Long lastModifiedBy;
    private LocalDateTime lastModifiedOn;
    private String apmosysRM;
    private String clientRm;
    private String apmosysRmEmail;
    private String poStartDate;
    private String poEndDate;
    private boolean activeFlag;
    private String prevPO;
    private String nextPO;
    private String msg;
    // private Long clientLocationId;

}
